### PR TITLE
Fresh deck links - fix header link styling and make the rerun url clickable

### PR DESCRIPTION
**Pull request by BenTheElder**
The first fix is purely stylistic but should look nicer (IE the "Prow Status" header will remain white despite being `a:visited` etc.). The second fix makes the Prow URL in the "re-run" job button clickable which is more convenient.

### CODE REVIEW

**Comment by rmmh** [comment]
don't you need to style a:visited?

**Comment by BenTheElder** [comment]
> don't you need to style a:visited?

no, I patched both of these in via the inspector before creating the changes. I also use this trick on my personal site.

**Comment by rmmh** [comment]
Ah, no, because it's a higher priority rule than the default stylesheet.

/unhold

File: prow/cmd/deck/static/script.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ function createRerunCell(modal, rerun_command, prowjob) {
     a.title = "Show instructions for rerunning this job.";
     a.onclick = function() {
         modal.style.display = "block";
-        rerun_command.textContent = "kubectl create -f \"" + url + "\"";
+        rerun_command.innerHTML = "kubectl create -f \"<a href='" + url + "'>" + url + "\"";
```
Comment (cjwagner) [inline prow/cmd/deck/static/script.js]: Closing tag?

Comment (BenTheElder) [reply prow/cmd/deck/static/script.js]: pfft [we don't need no stinking close tags](https://google.github.io/styleguide/htmlcssguide.html#Document_Type) /s
I'll fix that, heh.

**Comment by BenTheElder** [comment]
yeah, this way grabs up `a:*` just fine, [might possibly have issues with older browsers](https://stackoverflow.com/a/6722667).
/hold
to fix @cjwagner's comment

File: prow/cmd/deck/static/script.js (hunk 1 above)
Comment (BenTheElder) [reply prow/cmd/deck/static/script.js]: done.

**Pull request merged**